gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcSharedPolicyGroupHistoryRepository.java

```diff
@@ -78,17 +78,24 @@ protected JdbcObjectMapper<SharedPolicyGroup> buildOrm() {
 
     @Override
     protected String getId(SharedPolicyGroup item) {
-        return item.getId();
+        throw new IllegalStateException("Not implemented");
     }
 
     @Override
     public SharedPolicyGroup create(SharedPolicyGroup item) throws TechnicalException {
-        return super.create(item);
+        LOGGER.debug("JdbcSharedPolicyGroupHistoryRepository<{}>.create({})", getOrm().getTableName(), item);
+        try {
+            jdbcTemplate.update(buildInsertPreparedStatementCreator(item));
+            return getLatestBySharedPolicyGroupId(item.getEnvironmentId(), item.getId()).orElse(null);
+        } catch (final Exception ex) {
+            LOGGER.error("Failed to create SharedPolicyGroup item:", ex);
+            throw new TechnicalException("Failed to create SharedPolicyGroup item.", ex);
```

Comment:
Usually we don't place full stop in exception message?

Reply:
idem there is everything and no rule on this detail. but I'll delete it ;)